src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -161,11 +160,10 @@ public static class PrereleaseCompatibilityTransformer
                     // Files are now persisted to an array. We will persist a mapping from
                     // previous file key to file index. We will associate the index with
                     // each result when we iterate over run.results later.
-                    var files = run["files"] as JObject;                    
 
-                    if (files != null)
+                    if (run["files"] is JObject files)
```

Comment:
>run["files"] is JObject files [](start = 24, length = 29)

NIT: if you're going to introduce utilization of a new language pattern here, it is a common courtesy to scrub the code for other opportunities, rather than leaving things inconsistent. :) if you're not completely under the gun of course.